### ISSUE TITLE
Screen Reader: Many Pages: Non-interactive texts focused with Tab, have redundant semantics applied

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76937]
## Action Performed:
Prerequisite:
- User has an account and is logged in.

1. Using Windows Chrome + JAWS, open the page: [https://new.expensify.com/\](https://new.expensify.com//)
2. Tab to focus the Workspaces tab and press enter to activate.
3. Select any workspace and press enter to activate.
4. Tab to the 'Expensify Card' button and press enter to activate.
5. Tab through the resulting content and note what receives focus and is announced.

Other Occurrences:
1. On Book Travel

## Expected Result:
Only interactive elements should be focusable and assigned ARIA roles. Static text should not receive focus or use interactive semantics.

## Actual Result:
Static text “Unlimited virtual cards”, "Cash back on every US purchase", and "Spend controls and custom limits", receives keyboard focus and is exposed as a menuitem role element with an aria-label. This implies interactivity even though the element does not respond to any action.

HTML:
<div aria-label="Unlimited virtual cards" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-crgep1 r-17gur6a r-18u37iz r-1wtj0ep r-13qz1uu r-t60dpp r-1mkv55d r-1mdbw0j r-1e084wi r-gy4na3 r-9aemit r-wk8lta r-113qch9" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx r-13awgt0"><div class="css-g5y9jx r-18u37iz"><div class="css-g5y9jx r-eqz5dr r-13awgt0"><div class="css-g5y9jx r-18u37iz r-105ug2t"><div class="css-g5y9jx r-1awozwy r-1777fci" style="width: 40px;"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" id="simple-illustration__credit-cards_svg__Layer_1" x="0" y="0" viewBox="0 0 68 68" width="48" height="48" hovered="false" pressed="false"><style>.simple-illustration__credit-cards_svg__st1{fill:#002140}.simple-illustration__credit-cards_svg__st2{fill:#e96df2}</style><path ...</path></svg></div></div><div class="css-g5y9jx r-1777fci r-13awgt0" style="min-height: 40px;"><div class="css-g5y9jx r-18u37iz r-1awozwy r-dnmrzs"><div dir="auto" data-selection-scrapper-hidden-element="false" class="css-146c3p1" style="color: rgb(231, 236, 233); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; flex-shrink: 1; margin-left: 12px; white-space: pre-wrap; direction: ltr; max-width: 100%; line-height: 20px;">Unlimited virtual cards</div></div></div></div></div><div class="css-g5y9jx r-18u37iz" style="min-height: 40px;"></div></div></div></div>

## Area issue was found in:
Workspace Expensify Card

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Remove unnecessary interactive attributes such as tabindex="0", role="menuitem", and aria-label from non-interactive elements. If these are meant to be actionable, ensure they have a proper interactive role and corresponding functionality.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label

## Environment(s)
Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/65cc9765-5068-4baf-b646-f8f44b8971fa

![Image](https://github.com/user-attachments/assets/22b38cbd-d10d-4001-b064-8f5fe17892b5)

![Image](https://github.com/user-attachments/assets/d72d88d7-6825-4101-9f0d-6dd6b9bf9d0a)

![Image](https://github.com/user-attachments/assets/e422b1cc-fb6e-42d6-a552-d5480d3065ec)

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/7000159
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
done in https://github.com/Expensify/App/issues/79232
